docs/content/2.deploy/providers/firebase.md

```diff
@@ -20,19 +26,41 @@ This file should then be committed to version control. You can also create a `.f
 }
 ```
 
-Then, just add Firebase dependencies to your project:
+#### Install Firebase dependencies
+
+Then, add Firebase dependencies to your project:
+
+yarn:
 
 ```bash
-# yarn
 yarn add --dev firebase-admin firebase-functions firebase-functions-test
+```
 
-# npm
+npm:
+
+```bash
 npm install -D firebase-admin firebase-functions firebase-functions-test
 ```
 
-### Using Firebase CLI
+#### Install Firebase CLI globally
+
+Make sure you have the latest version of the firebce CLI installed:
+
+yarn:
+
+```bash
+yarn global add firebase-tools
+```
+
+npm:
+
+```bash
+npm install -g firebase-tools
```

Comment:
We might instead use `npx firebase-tools@[version]`

Reply:
Have updated the docs in [Change docs to recommend using npx firebase-tools](https://github.com/unjs/nitro/pull/1142/commits/424fe2ddecc0d37a2485a6b1ea9e2759419c3300)